test/fixtures/wallet/wallet.js

```diff
@@ -0,0 +1,29 @@
+/* Import fixtures
+ *
+ * Sample wallet for tests
+ *
+ * @return {Object}
+ */
+
+const currencies = [
+  'neelam',
+  'dinero'
+]
```

Comment:
Better to have these imported from `constants`?

Reply:
removing them since not used. Added for future purposes